src/server/game/Entities/Vehicle/Vehicle.cpp

```diff
@@ -785,6 +785,11 @@ bool VehicleJoinEvent::Execute(uint64, uint32)
         return true;
     }
 
+    //It's possible that multiple vehicle join
+    //events are executed in the same update
```

Comment:
Cosmetic detail, but seeing the length of the code before & after this addition, maybe the comment could be reduced to 1 line?

Reply:
What I want to suggest here is both to drag back line 2 (789) into line 1 (788) and to add a blank space after the comment backslashes (`//`) like this:
```cpp
    // It's possible that multiple vehicle join events are executed in the same update
```
( I simply thought that this format would follow the code style standard a bit more.)